examples/server/grammar.py

```diff
@@ -0,0 +1,99 @@
+from openai import OpenAI
+
+client = OpenAI(api_key="foobar", base_url="http://localhost:1234/v1/")
+
+JSON_KBNF = '''
+(* JSON Grammar *)
+
+(* JSON text must contain a single JSON value *)
+start       = value ;
+
+(* A JSON value can be an object, array, string, number, true, false, or null *)
+value      ::= object 
+           | array 
+           | string 
+           | number 
+           | "true" 
+           | "false" 
+           | "null" ;
+
+(* A JSON object is a collection of key/value pairs enclosed in curly braces *)
+object     ::= "{" [ members ] "}" ;
+members    ::= pair { "," pair } ;
```

Comment:
You can use both `{members}` and `members*` to express 0 or infinite repetition.